lib/sles4sap_publiccloud.pm

```diff
@@ -921,14 +928,18 @@ sub create_playbook_section_list {
     my @playbook_list;
 
     unless ($args{registration} eq 'noreg') {
-        # Add registration module as first element
-        my $reg_code = '-e reg_code=' . get_required_var('SCC_REGCODE_SLES4SAP') . " -e email_address=''";
-        if ($args{registration} eq 'suseconnect') {
-            push @playbook_list, "registration.yaml $reg_code -e use_suseconnect=true";
-        }
-        else {
-            push @playbook_list, "registration.yaml $reg_code";
+        my @reg_args = ('registration.yaml');
```

Comment:
as the command line composition for this playbook become more complex I decided to fit all argument in a list and then join them at the end